include/ocpp/common/cistring.hpp

```diff
@@ -23,6 +23,12 @@ template <size_t L> class CiString : public String<L> {
     CiString(const char* data) : String<L>(data) {
     }
 
+    CiString(json data) : String<L>(data.get<std::string>()) {
```

Comment:
I'm not sure about having this conversion implicit since it is quite a big step from json to a CiString.

Reply:
You are right, I removed this constructor again because it's a bit too drastic